indico/web/client/js/react/components/WTFPrivacyPolicyURLsField.jsx

```diff
@@ -0,0 +1,108 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2022 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+
+import {Translate} from 'indico/react/i18n';
+
+export default function WTFPrivacyPolicyURLsField({fieldId, fields, initialValues}) {
+  const [values, setValues] = useState([]);
+  const tempValues = initialValues;
```

Comment:
This looks strange...

- you'll mutate initialValues when mutating tempValues i addURL
- you change local state without re-rendering

Reply:
Yeah, I see the problem, I'm still figuring this out. Thank's for the tips!